Assets/Decontainment/Singletons/BotManager.cs

```diff
@@ -71,4 +75,14 @@ public int GetTargetHeading(Controller targeter, int targetIndex)
         Vector2 look = target.transform.position - targeter.transform.position;
         return (int)Vector2.SignedAngle(targeter.transform.right, look);
     }
+
+    public IEnumerator PropagationCoroutine (int registerNumber, int registerValue, int botTeamID) {
```

Comment:
```suggestion
    public IEnumerator PropagationCoroutine(int registerNumber, int registerValue, int botTeamID)
    {
```
Stay consistent with the rest of the codebase